Update project README (#21825)
wordpress-mobile/WordPress-iOS: README.md

# WordPress for iOS #

## Build Instructions

Please refer to the sections below for more detailed information. The instructions assume the work is performed from a command line.

### Getting Started

> 1. [Download](https://developer.apple.com/downloads/index.action) and install Xcode. *WordPress for iOS* requires Xcode `14.3.1` or newer.

Reviewer: A spelled out Xcode version in the README is bound to become out of sync, like the diff shows.

What do you think of mentioning the `.xcode-version` file?

https://github.com/wordpress-mobile/WordPress-iOS/blob/05a954e50ad82c1e651ce0e972bcb8dadc5c7be6/.xcode-version

```suggestion
1. [Download](https://developer.apple.com/downloads/index.action) and install Xcode. Refer to the `.xcode-version` file for the minimum required version.
```

Author: Good call! One less thing to maintain. 👍

1. [Clone this repository](https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository) in the folder of your preference.

> 1. In a terminal, enter the `WordPress-iOS` directory where it was cloned.

Reviewer: Is this line necessary?

From one point of view, it's good not to make assumptions. Then again, it should be a given that once you clone a repo, you go into that folder to work on it..

Author: Good call, tweaked in https://github.com/wordpress-mobile/WordPress-iOS/pull/21825/commits/3e7ceeca8dbd2e2dbb04497628926659d221297c.

At the risk of being verbose I was thinking about the users who clone via GUI tools and needed to be in the terminal at that point. 👍
